webview-ui/src/App.tsx

```diff
@@ -20,7 +20,7 @@ import { vscode } from "./utils/vscode";
 import GuidedApproachWizard from "./components/GuidedApproachWizard";
 import ProgressIndicator from "./components/ProgressIndicator";
 import ViolationIncidentsList from "./components/ViolationIncidentsList";
-import { Incident, RuleSet } from "./types";
+import { Incident, RuleSet } from "@shared/types";
```

Comment:
Oversight on my end removing the tsc compilation step from the webview build script.